### PR TITLE
WIP: Add new fixtures & fix test_activation tests

### PR DESCRIPTION
**Pull request by Dostonbek1**

### CODE REVIEW

**Comment by Dostonbek1** [comment]
@AlanCoding This is still in progress. I need to fine-tune some more parts of test_activation.py but just wanted to show where I'm going with this. Currently, everything is passing in `test_activation.py` tests. 
```
tests/integration/api/test_activation.py .............................                                                                                  [100%]
...
115 failed, 338 passed, 24 skipped, 3 warnings
```

File: tests/integration/api/conftest.py [hunk 1]
```diff
@@ -77,12 +86,13 @@ def check_permission_mock():
...
 
 @pytest.fixture
-def default_de() -> models.DecisionEnvironment:
+def default_de(default_organization) -> models.DecisionEnvironment:
     """Return a default DE."""
     return models.DecisionEnvironment.objects.create(
         name="default_de",
         image_url="quay.io/ansible/ansible-rulebook:latest",
         description="Default DE",
+        organization=default_organization,
```
Comment (AlanCoding) [inline tests/integration/api/conftest.py:95]: This is fine, but for the record, you should get the same behavior if you use the `default_organization` as a fixture but _don't_ pass it as a kwarg here.

Comment (Dostonbek1) [reply tests/integration/api/conftest.py:95]: Ah makes sense. I'll remove the args then.

Comment (Dostonbek1) [reply tests/integration/api/conftest.py:95]: Actually, I should keep them because tests started failing after I removed them.

**Pull request merged**